recipes/bob.io.matlab/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "2.0.4" %}
+
+package:
+  name: bob.io.matlab
+  version: {{ version }}
+
+source:
+  fn: bob.io.matlab-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.io.matlab/bob.io.matlab-{{ version }}.zip
+  md5: 089155b2d864a6dca0f30491da0ed4fc
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Looking at other `bob` packages, I do not see any `.pyc` files which means `-B` works as intended.
I added the `-B` option since `conda` was failing to compile `bob` packages with Python 3. It was complaining that the `__py_cache__` (something like that) folder already exists.